experiments/fire.rst

```diff
@@ -0,0 +1,95 @@
+
+===========
+Making fire
+===========
+
+  Name:: Making fire from primitive beginnings
+
+  History::
+        You could argue that the ability to make fire was the discovery
+        that really put humans near the top of the animal survival charts.
+        Along with sourcing clean water, clothing and shelter, it's
+        considered a useful survival skill, and a great way to demonstrate
+        some basic principles.
+
+  Age group::
+        5+, but with adult supervision - there's cutting and fire involved
+
+  Requirements::
+        Access to small bits of dry timber
+        A small round bit of dry timber (like a dowel rod)
+        Some cotton wool (makes great kindling)
+        A green (fresh) stick from a tree, anywhere between 30cm and 60cm long (TODO : revise)
+        Some fibrous material to make a cord from (or rope, if you want to cheat)
+        Somewhere safe and outdoors to start the fire (concrete driveway is fine)
+        A knife (pocket knives are great)
+
+  Warnings::
+        This is literally playing with fire.  Do it outside, in a clean well ventilated area.
+        Have a bucket of water or sand on standby, and make sure the area is away from combustible
+        materials.  A fire extinguisher isn't recommended, due to the high pressures involved being able to blow
+        flaming combustion around - better to douse in sand or water.
+        Make sure there's no large winds or other factors that might precipitate disaster.
+        If you've never made a campfire before, practice first, to get an idea of how to stack kindling
+        to make a fire, and how to properly contain a small fire.
+
+  Options::
+        Consider first demonstrating with matches, to get the safety protocol clear.
+        Also consider demonstrating some alternative methods, which are spelled out below (TODO : spell out below)
+        It can be helpful to make a fun goal at the end to stimulate interest (I recommend toasting a marshmallow!)
+
+  Method::
+        Safety first, make sure the area you're making fire in is clear from debris, away from combustible materials.
+
+        A basic bow drill is using a rapidly spinning bit of wood (the drill) to cause a huge friction in another
+        part of wood, such that an ember is formed.
+        Transferring the ember to kindling and applying air flow (blowing on it) is how the fire is actually made.
+
+        Preparing your materials::
+            Take the green stick and bend it to form a semicircular bow-like shape.
+
+            Fix the cord to either end, such that the stick can no longer go straight,
+            but not so tight that you could use it to fire an arrow.  A little slack is what you're after.
+
+            Take the dowel, and ensure the ends are rounded and not square, by trimming it a bit with the knife.
+            It doesn't have to be perfect, you're just looking to remove the right angles
+            and make it fit easier into the joints.
```

Comment:
I recommend one thick round end, and one pointy end.  THink a big fat pencil  THe rounded end is the bottom, getting hot end, the pointy end is the top where you will need green leaves or grass as watery lubricant.